docs/api.rst

```diff
@@ -0,0 +1,14 @@
+API Docs
```

Comment:
Why is this file called `api.rst` and not `modules.rst`?

Reply:
So this was done for the sole purpose of having the API Docs title in the sidebar and in the index page at the bottom (the api.rst connects to the module.rst). I agree this may not merit an additional rst file. 
I was following the queenbee approach though I think that library is very unique in itself, so we can keep it simpler here and avoid the api.rst file if you prefer.
Below is how it will look like, Basically I generated the doc index page with the module.rst instead of the api.rst.

![image](https://user-images.githubusercontent.com/19495783/87090579-3865a800-c206-11ea-85ce-2694c255ef0a.png)

Reply:
I see now. That landing page looks beautiful and using `api.rst` is a good idea.